README.md

```diff
@@ -46,10 +46,12 @@ A curated list of awesome tools, platforms, products, and resources for Managed
 - [Autotask PSA](https://www.datto.com/products/autotask-psa) - Cloud solution offers ITL-aligned ticketing model, open APIs, ITSM tools, integrations with Datto RMM and more.
 - [ConnectWise Manage](https://www.connectwise.com/software/manage) - MSP PSA software with integrated help desk, sales & marketing, procurement, time tracking & billing.
 - [Freshdesk](https://freshdesk.com/) - cloud-based customer service software that gives you everything you need to deliver support.
+- [HaloPSA](https://halopsa.com/) - All-in-One PSA Software for Managed Service Providers.
 - [Kaseya BMS](https://www.kaseya.com/products/bms/) - Business management solution.
 - [ManageEngine ServiceDesk Plus](https://www.manageengine.com/products/service-desk/) - IT service management (ITSM) software with integrated IT asset management, CMDB, problem management, and change and release management, along with service desk capabilities.
 - [Tigerpaw](https://www.tigerpaw.com/) - End-to-end business automation and management across all aspects of your business.
 - [Zendesk](https://www.zendesk.com/) - Customer service and support ticketing system.
+- [ZestMSP])https://www.zestmsp.com/) - MSP PSA software with integrated help desk, time tracking, and more.
```

Comment:
Please fix the left parenthesis otherwise the link won't work :)

Reply:
Haha, wow, not my day. Sorry, code reviews for the win!